maintainers/maintainer-list.nix

```diff
@@ -4404,6 +4404,12 @@
       fingerprint = "8992 44FC D291 5CA2 0A97  802C 156C 88A5 B0A0 4B2A";
     }];
   };
+  kiyengar = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add kiyengar``.